batch/src/test/java/org/openmrs/analytics/JdbcFetchUtilTest.java

```diff
@@ -131,16 +131,31 @@ public void testCreateSearchSegmentDescriptor() {
 	}
 	
 	@Test
-	public void testCreateFhirReverseMap() throws IOException {
-		// here we pass Encounters as such we expect visits to be included in the reverseMap as well
-		Map<String, String> reverseMap = jdbcFetchUtil.createFhirReverseMap("Patient,Encounter,Observation",
+	public void testCreateFhirReverseMap() throws Exception {
+		Map<String, String> reverseMap = jdbcFetchUtil.createFhirReverseMap("Patient,Person,Encounter,Observation",
 		    "../utils/dbz_event_to_fhir_config.json");
-		// we expect 4 objects, and visit should be included
-		assertEquals(4, reverseMap.size());// not 3
-		assertFalse(reverseMap.get("visit").isEmpty());
-		assertFalse(reverseMap.get("encounter").isEmpty());
-		assertFalse(reverseMap.get("obs").isEmpty());
-		assertFalse(reverseMap.get("person").isEmpty());
+		assertEquals(4, reverseMap.size());
+		assertEquals(reverseMap.get("Patient"), "person");
+		assertEquals(reverseMap.get("Person"), "person");
+		assertEquals(reverseMap.get("Encounter"), "encounter");
```

Comment:
So specifically here, `Encounter` should be mapped to both `encounter` and `visit`.

Reply:
corrected in the new commit